main.go

```diff
@@ -244,6 +253,8 @@ func parseFlags(logger log.Logger) (config, error) {
 		"The address on which public server runs.")
 	flag.StringVar(&cfg.server.listenInternal, "web.internal.listen", ":8081",
 		"The address on which internal server runs.")
+	flag.StringVar(&cfg.server.healthcheckURL, "web.healthchecks.url", "http://localhost:8080",
+		"The URL like http://localhost:8080 on which public server runs and to run healthcheck tests against.")
```

Comment:
I think we can leave out `like http://localhost:8080`, since URL says it all already.
Also, nit: the other flag descriptions don't end in a `.`. I prefer periods but let's choose one.

```suggestion
		"The URL at which the public server runs and against which to run healthcheck tests ")
```

Reply:
All usages end with a period. :thinking: So does mine?!

Reply:
gah yes you were right, i must have mis-read :see_no_evil: